### PR TITLE
January Cleaning 

### PR DESCRIPTION
**Pull request by tigercat2000**
rawr

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -97,6 +97,7 @@
 // ++++ROCKDTBEN++++ MOB PROCS -- Ask me before touching.
 // Stop! ... Hammertime! ~Carn
 // I touched them without asking... I'm soooo edgy ~Erro (added nodamage checks)
+// no ~Tigerkitty
```
Comment (TheDZD) [inline code/modules/mob/living/living.dm]: Dafuq oldcoders?

Comment (tigercat2000) [inline code/modules/mob/living/living.dm]: god knows

**Comment by Fox-McCloud** [comment]
>++ is like the only single consistent thing in our entire codebase. It should be stuck to.

->proves that isn't the case with this PR.

**Comment by tigercat2000** [comment]
@Fox-McCloud rawr

**Pull request merged**